code/modules/cargo/bounties/atmos.dm

```diff
@@ -0,0 +1,42 @@
+/datum/bounty/item/atmospherics
+	name = "Full Tank of Pluoxium"
+	description = "CentCom RnD is researching extra compact internals. Ship us a tank full of Pluoxium and you'll be compensated."
```

Comment:
For consistency, have this one include the "(20 Moles)" that the other lines have, as well.

Reply:
Done